timesketch/lib/analyzers/sequence_sessionizer.py

```diff
@@ -0,0 +1,152 @@
+"""Sequenced activity sessionizing sketch analyzer plugin."""
+
+from __future__ import unicode_literals
+
+#from timesketch.lib.analyzers import manager
+from timesketch.lib.analyzers import sessionizer
+
+
+class SequenceSessionizerSketchPlugin(
+        sessionizer.SessionizerSketchPlugin):
+    """Base class for specialized sequenced activity sessionizings
+    sketch plugins.
+
+    Attributes:
+        event_seq_name: The name of the event sequence.
+        event_seq: List of dictionaries of attributes describing the events
+        event_storage: List of events.
+        num_event_to_find: Number that shows which event from the sequence
+        should be matched.
+        recording: Shows if currently are finded and stored events that match
+        the specified sequence of events.
+        return_fields: List of name of event attributes, should be specified
+        in the inheriting sessionizers. It's mandatory to contains 'timestamp'.
+        session_num: Counter for the number of sessions.
+    """
+    event_seq_name = None
+    event_seq = None
+    event_storage = []
+    num_event_to_find = 0
+    recording = False
+    return_fields = ['timestamp']
+    session_num = 0
+
+    def run(self):
+        """Entry point for the analyzer. Allocates each event between the
+        first event of the event_seq and the last event of the event_seq
+        an event_seq_name attribute and a session_num.
+
+        Returns:
+            String containing the name of the event sequence and the
+            number of sessions created.
+        """
+        if self.event_seq_name is None:
+            raise RuntimeError('No event_seq_name provided.')
+        if self.event_seq is None or self.event_seq == []:
+            raise RuntimeError('No event_seq provided.')
+        if 'timestamp' not in self.return_fields:
+            raise RuntimeError('No \'timestamp\' in return_fields.')
```

Comment:
Alternatively, you can use double quotes around `"timestamp"` to avoid escaping single quotes